client/web/web.go

```diff
@@ -756,6 +732,53 @@ func (s *Server) tailscaleUp(ctx context.Context, st *ipnstate.Status, postData
 	}
 }
 
+type tailscaleUpOptions struct {
```

Comment:
One concern with that is then I'm not sure how to interpret various combinations of those values.  What does it mean if both Reauthenticate and Connect are true?  What about both false?  I'll add docs to make clear that if Reauthenticate is false, then it will simply connect.  I also like the analog the running `tailscale up` from the CLI with no parameters... in that case it just reconnects.  So I like calling `/api/up` with no options having the same semantics.

Reply:
Oh didn't realize that was the CLI behavior. In that case I don't mind it! Maybe add a little doc